libs/remix-ui/home-tab/src/lib/components/homeTabGetStarted.tsx

```diff
@@ -30,31 +30,31 @@ const workspaceTemplates: WorkspaceTemplate[] = [
   {
     gsID: 'sUTLogo',
     workspaceTitle: 'Start Coding',
-    description: 'Create a new project using this template.',
+    description: 'Start coding using the default template.',
     projectLogo: 'assets/img/remixverticaltextLogo.png',
     templateName: 'remixDefault',
   },
   {
     gsID: 'sUTLogo',
-    workspaceTitle: 'Circom',
+    workspaceTitle: 'Zk Semaphore',
```

Comment:
Please make it:
ZK Semaphore
(capital K)

Reply:
done